src/sig_stfl/sig_stfl.h

```diff
@@ -433,7 +447,7 @@ OQS_API OQS_STATUS OQS_SIG_STFL_sign(const OQS_SIG_STFL *sig, uint8_t *signature
 OQS_API OQS_STATUS OQS_SIG_STFL_verify(const OQS_SIG_STFL *sig, const uint8_t *message, size_t message_len, const uint8_t *signature, size_t signature_len, const uint8_t *public_key);
 
 /**
- * Query number of remaining signatures
+ * Query the number of remaining signatures.
```

Comment:
"remain" undocumented?

Reply:
Added the description why the remaining signature and total signature matters in https://github.com/open-quantum-safe/liboqs/commit/c577c2ed8d6597c1766f5d9612397ed876061aa3